src/test/test_translator_drone.py

```diff
@@ -0,0 +1,24 @@
+from click.testing import CliRunner
+
+from popper import __version__
+from popper import _version_file
+from popper.commands import cmd_version
+from popper.translators.translator_drone import DroneTranslator
+from .test_common import PopperTest
+
+
+class TestDroneTranslator(PopperTest):
```

Comment:
Can we also add tests for `translate_step` and `translate` ?

Reply:
resolved 

https://github.com/getpopper/popper/pull/976/files#diff-59acabdf0d01eb06086a11bb0f995aab6e2939f239067b1dc9c8358b428ca6cfR12

https://github.com/getpopper/popper/pull/976/files#diff-59acabdf0d01eb06086a11bb0f995aab6e2939f239067b1dc9c8358b428ca6cfR50

Reply:
Thanks